test/e2e/common_suite_test.go

```diff
@@ -521,3 +624,11 @@ func doTestCreatePeerPodAndCheckEnvVariableLogsWithImageAndDeployment(t *testing
 	expectedPodLogString := "ISPRODUCTION=true"
 	newTestCase(t, "EnvVariablePeerPodWithBoth", assert, "Peer pod with environmental variables has been created").withPod(pod).withExpectedPodLogString(expectedPodLogString).withCustomPodState(v1.PodSucceeded).run()
 }
+
+func doTestCreatePeerPodWithLargeImage(t *testing.T, assert CloudAssert) {
+	namespace := envconf.RandomName("default", 7)
+	podName := "largeimage-" + strconv.Itoa(rand.Intn(1000)) + "-pod"
```

Comment:
Why does the pod name of the large image pod contain a random substring?

Reply:
Hi @mattarnoatibm ,
Initially we assessed the pod completion status based on event logs of pod. In order to avoid repeated events during assessment, we used to generate pod with different name each time. Now we changed our approach in assessment by checking the Logs of CAA. Will remove the random substring from podName now. Thanks!